### PR TITLE
[OT276-23] Crear modelo y repositorio base de Members

### PR DESCRIPTION
**Pull request by matiaguro**
## Why is this change required?
COMO desarrollador 
QUIERO agregar la entidad Member
PARA representar en la implementación la estructura de datos

Criterios de aceptación: 
Nombre de tabla: members. Campos:
name: VARCHAR NOT NULL
facebookUrl: VARCHAR NULLABLE
instagramUrl: VARCHAR NULLABLE
linkedinUrl: VARCHAR NULLABLE
image: VARCHAR NOT NULL
description: VARCHAR NULLABLE
timestamps y softDelete
### Type of change
- [ x] New feature
- [ ] Bug fix
- [ ] New Tests (no functional change)
- [ ] Refactor (no functional change)

### Checklist

- [ x] Traceability between this change and Jira.
- [ ] ```mvn clean install``` was run and all tests completed successfully.
- [ ] There are no unused variables and/or imports in the modified classes.
- [ ] There are no imports in the modified classes with the wildcard character. Ex: ```com.somepackage.*```.
- [ ] Slf4j was used for the application log instead ```System.out```.
- [ ] Public methods are documented with ```javadoc```.

### CODE REVIEW

File: src/main/resources/db/migration/V1.00.1__create_table_member.sql [hunk 1]
```diff
@@ -0,0 +1,17 @@
+drop table if exist member;
```
Comment (manerajona) [inline src/main/resources/db/migration/V1.00.1__create_table_member.sql]: Por favor cambiar el nombre del script a: V1.23__create_table_member.sql

Comment (manerajona) [inline src/main/resources/db/migration/V1.00.1__create_table_member.sql]: Por favor utilizar la sentencia `exists`, exist no existe.

File: src/main/resources/db/migration/V1.00.1__create_table_member.sql [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+create table member
+
+    {
```
Comment (manerajona) [inline src/main/resources/db/migration/V1.00.1__create_table_member.sql]: No se utilizan llaves al crear scripts sql, usar paréntesis `()`

File: src/main/resources/db/migration/V1.00.1__create_table_member.sql [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    member_id       bigint         not null auto_increment,
+    name            varchar(255)   not null,
+    facebookUrl     varchar(255)   nullable,
```
Comment (manerajona) [inline src/main/resources/db/migration/V1.00.1__create_table_member.sql]: Remover todas la sentencias `nullable` por favor

File: src/main/resources/db/migration/V1.00.1__create_table_member.sql [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+    instagramUrl    varchar(255)   nullable,
+    linkedinUrl     varchar(255)   nullable,
+    image           varchar(255)   not null,
+    description     varchar(255)   nullable,
+    created_at      datetime(6)    not null,
+    is_active       bit            not null,
+    updated_at      datetime(6)    null
```
Comment (manerajona) [inline src/main/resources/db/migration/V1.00.1__create_table_member.sql]: falta una coma al final del statement

File: src/main/resources/db/migration/V1.00.1__create_table_member.sql (hunk 3 above)
Comment (manerajona) [inline src/main/resources/db/migration/V1.00.1__create_table_member.sql]: Las columnas de la tabla van en snake_case no camelCase.

**Review by manerajona — CHANGES_REQUESTED** [review]
Por favor probar que levante la aplicación @matiaguro

**Comment by matiaguro** [comment]
Se corrigieron los errores marcados

**Comment by matiaguro** [comment]
@manerajona

**Pull request merged**